scope_extensions-explainer.md

```diff
@@ -70,13 +90,16 @@ associated origins.
    }
    ```
 
-1. Let the extended scope of a web app be the set of URLs that:
-    - Has an origin that matches one of the origin patterns in the manifest's
-      `scope_extensions` list.
-    - Has an origin with a valid
-      `<origin>/.well-known/web-app-origin-association` association file
-      with an association entry matching the web app's
-      [identity](manifest-identity).
+The `web_app_identity` field must contain a valid [web application
+id](https://w3c.github.io/manifest/#id-member).
+
+### Resulting extended scope
+Let the extended scope of a web app be the set of URLs that:
```

Comment:
rephrase maybe as:

A url is in the extended scope of a web app if both:

* It matches an entry in the `scope_extensions` list.
* That entry has been validated by fetching the `<origin>/.well-known/web-app-origin-association` association file with an entry matching the app's [identity](manifest-identity).

Reply:
Done.